verilog/analysis/checkers/no_trailing_spaces_rule.cc

```diff
@@ -53,13 +53,18 @@ const LintRuleDescriptor& NoTrailingSpacesRule::GetDescriptor() {
 }
 
 void NoTrailingSpacesRule::HandleLine(absl::string_view line) {
+  // Lines may end with \n or \r\n. '\n' is already excluded.
+  // Exclude '\r'
+  if (!line.empty() && line.back() == '\r') {
+    line.remove_suffix(1);
+  }
+  // Now any line endings (either \n or \r\n) are excluded.
   // Searches each line in reverse for spaces.
   const auto rbegin = line.crbegin();
   const auto rend = line.crend();
   if (rbegin != rend) {
-    // Lines already exclude \n, so we can check using std::isspace.
     const auto reverse_iter =
-        std::find_if(rbegin, rend, [](char c) { return !std::isspace(c); });
+        std::find_if(rbegin, rend, [](char c) { return !std::isblank(c); });
```

Comment:
After you've removed the `\r` above, do you still need the change from isspace to isblank ? I suspect now the original code will just work.

Reply:
Yes, definitely the previous one will be better as it matches a narrower set of characters. It's `isspace` again.